homeassistant/components/tuya/select.py

```diff
@@ -338,6 +338,15 @@
             translation_key="blanket_level",
         ),
     ),
+    # Smart Odor Eliminator-Pro
+    # Undocumented, see https://github.com/orgs/home-assistant/discussions/79
+    "cwjwq": (
+        SelectEntityDescription(
+            key=DPCode.WORK_MODE,
+            entity_category=EntityCategory.CONFIG,
+            translation_key="work_mode",
```

Comment:
What would this be for this device?

This sounds like data point with a different naming/purpose in the UI.

Reply:
> What would this be for this device?
> 
> This sounds like data point with a different naming/purpose in the UI.

I have updated the strings